tools/compiletest/src/main.rs

```diff
@@ -331,11 +331,19 @@ pub fn make_tests(config: &Config, tests: &mut Vec<test::TestDescAndFn>) {
 
 /// Returns a stamp constructed from input files common to all test cases.
 fn common_inputs_stamp(config: &Config) -> Stamp {
-    let mut stamp = Stamp::from_path(&config.kani_dir_path);
-
-    // Add compiletest itself.
     let rust_src_dir = config.find_rust_src_root().expect("Could not find Rust source root");
+    let kani_bin_path = &rust_src_dir.join("target/debug/kani-compiler");
```

Comment:
It works the same way - if there are changes in any of the other directories, it will re-run the tests.

Note that `Stamp::from_path` basically works as a constructor here:
```rust
impl Stamp {
    fn from_path(path: &Path) -> Self {
        let mut stamp = Stamp { time: SystemTime::UNIX_EPOCH };
        stamp.add_path(path);
        stamp
    }
[...]
}
```

I tried passing `target/*/kani-compiler` but that does not seem to work. So there are a few options here:
 * Create a `new` constructor that returns `Stamp { time: SystemTime::UNIX_EPOCH }`, not looking at any binaries.
 * Add also the path to the release binary.
But it would not make a big difference, test re-runs will almost always be triggered by changes in the folders below.

Reply:
I guess we can leave it like that for now. Thanks

Reply:
I just wanted to make sure it wouldn't fail to run the tests.